de/django_forms/README.md

```diff
@@ -237,7 +249,7 @@ Füge dies direkt am Anfang der Datei hinzu. Jetzt können wir endlich sagen: "G
 return redirect('post_detail', pk=post.pk)
 ```
 
-`post_detail` ist der Name unserer View, zu der wir springen wollen. Erinnerst du dich, dass diese *View* einen `pk` benötigt? Um diesen an die View weiterzugeben, benutzen wir `pk=post.pk`, wobei `post` unser neu erstellter Blogpost ist!
+`post_detail` ist der Name der View, zu der wir springen wollen. Erinnerst du dich daran, dass dieser *view* einen `pk` benötigt? Um diesen an die View weiterzugeben, benutzen wir `pk=post.pk`, wobei `post` unser neu erstellter Blogpost ist!
```

Comment:
I'm use to "View" having a feminine grammatical gender in German. That seems to also be how we've used it until now in throughout the German translation of the Django Girls tutorial **and** in the first sentence of this paragraph ("der Name der View" &mdash; male or neutral would be "der Name **des** View**s**") and in the last one, too ("die View" instead of "das View" or "der View").

```suggestion
`post_detail` ist der Name der View, zu der wir springen wollen. Erinnerst du dich daran, dass diese *view* einen `pk` benötigt? Um diesen an die View weiterzugeben, benutzen wir `pk=post.pk`, wobei `post` unser neu erstellter Blogpost ist!
```

Reply:
changed on Crowdin